drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableSortGridWidgetMouseEventHandler.java

```diff
@@ -42,12 +43,18 @@ public boolean handleHeaderCell(final GridWidget gridWidget,
         final BaseGridRendererHelper rendererHelper = gridWidget.getRendererHelper();
         final BaseGridRendererHelper.ColumnInformation ci = rendererHelper.getColumnInformation(relativeLocation.getX());
         final GridColumn<?> column = ci.getColumn();
-        if (column == null) {
-            sortCallback.onFailure(null);
-            return false;
-        }
-        sortCallback.onSuccess(column);
+        sortConsumer.accept(column);
 
         return true;
     }
+
+    @Override
+    public boolean handleBodyCell(final GridWidget gridWidget,
+                                  final Point2D relativeLocation,
+                                  final int uiRowIndex,
+                                  final int uiColumnIndex,
+                                  final AbstractNodeMouseEvent event) {
+        // sorting is done by header cells, do not handle body cells
+        return false;
+    }
```

Comment:
I am sorry for the inconvenience. As discussed privately. This needs to be removed. Otherwise cells can not be edited by mouse.

Reply:
No worries

Reply:
Removed